### PR TITLE
change defaults and default smoothing for dynamic monitor

### PR DESCRIPTION
**Pull request by michaelneale**
Given little is known about the data - some more sensible defaults and smoothing is a good idea.

### CODE REVIEW

File: monitor/run_monitor_dyn.py [hunk 1]
```diff
@@ -41,7 +46,8 @@ def get_monitor(check_id, config):
 
     if check_id not in current_monitors:
         current_monitors[check_id] = new_monitor(check_id, config)
-    return current_monitors[check_id]
+        moving_averages[check_id] = deque([0.0] * 10, maxlen=10)
```
Comment (allanino) [inline monitor/run_monitor_dyn.py]: Hey @michaelneale, I think it's better to get the moving average window from the config, defaulting it to `10` if you'd like. I think something like this should work even some config sections are absent:
```
moving_average_window = int(config.get('parameters', {'moving_average_window': 10}).get('moving_average_window', 10))
moving_averages[check_id] = deque([0.0] * moving_average_window, maxlen=moving_average_window)
```

Comment (michaelneale) [inline monitor/run_monitor_dyn.py]: agree thats a good idea, will do that.

**Comment by michaelneale** [comment]
OK updated as recommended.

**Comment by allanino** [comment]
Great! Thanks!

**Pull request merged**